packages/commonwealth/server/migrations/20230411172411-cosmos-v1-sandbox.js

```diff
@@ -0,0 +1,56 @@
+'use strict';
+
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    await queryInterface.sequelize.transaction(async (t) => {
+      await queryInterface.sequelize.query(
+        `
+        INSERT INTO "ChainNodes" (name, url, alt_wallet_url, balance_type, bech32) VALUES (
+          'cosmos sdk devnet',
+          'http://ec2-35-173-213-72.compute-1.amazonaws.com:26657',
+          'http://ec2-35-173-213-72.compute-1.amazonaws.com:1317',
```

Comment:
@NakulManchanda we will need these to be hosted in `https` to run in staging environments.

Reply:
https://ec2-35-173-213-72.compute-1.amazonaws.com/lcd -> http://ec2-35-173-213-72.compute-1.amazonaws.com:1317

https://ec2-35-173-213-72.compute-1.amazonaws.com/rpc -> http://ec2-35-173-213-72.compute-1.amazonaws.com:26557

self signed cert though try with these

- if self signed cert won’t work we can get real one too